_search-plugins/inner-hits.md

```diff
@@ -0,0 +1,750 @@
+---
+layout: default
+title: Inner hits
+has_children: false
+nav_order: 215
+---
+
+
+
+# Inner_hits
+
+In OpenSearch, when you perform a search using parent-join or nested objects, the underlying hits that resulted in the hits being returned (child documents or nested inner objects) are hidden by default. Whether you are using parent/child, nested objects or both, there could different reasons why retrieving these underlying hits is important. This is achieved using `inner_hits` parameter in the search query.
```

Comment:
should we add a hyper link to - https://opensearch.org/docs/latest/field-types/supported-field-types/join/

Reply:
@rishabhmaurya thank you for review and comments, thats updated now